patches/tools-resources-generate_resource_allowlist.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/resources/generate_resource_allowlist.py b/tools/resources/generate_resource_allowlist.py
+index 94ea65ea253a49b204935f2fbdfe17cf8fb83a6a..d30024556d3de79165814175585dc532494ce34f 100755
+--- a/tools/resources/generate_resource_allowlist.py
++++ b/tools/resources/generate_resource_allowlist.py
+@@ -136,7 +136,7 @@ def WriteResourceAllowlist(args):
+     resource_ids.update(func(input))
+ 
+   # The last time this broke, exactly two resources were still being found.
+-  if len(resource_ids) < 100:
```

Comment:
```The error message is actually misleading due to a bug in the upstream
script (trying to use string.join on int numbers), but the code where
this error happens is:```
can we submit a fix?

Reply:
Just to fix the `join`? I don't see why not, but we'd still need the patch to adjust the number. I am not sure of upstream will be interested in lowering the number or not - it's arbitrary from what I can tell.